GetAPIData.py

```diff
@@ -18,14 +19,22 @@
 
 session = Session()
 session.headers.update(headers)
+apiCoins = []
 
 @app.route('/')
 @app.route('/coins')
 def coinsHome():
     try:
         response = session.get(url, params=parameters)
         apiData = json.loads(response.text)
-        return render_template('coin.html', data=apiData['data'])
+        coinHeader =  Coin("Coin", "Name","Price", "Market Cap")
+        for c in apiData['data']:
+          coinData = Coin(c['name'],
+          c['symbol'],
+          c['quote']['CAD']['price'],
+          c['quote']['CAD']['market_cap'])
```

Comment:
You may want to reformat it in a more standard way

Reply:
updated